testsuite/python/lb_pressure_tensor.py

```diff
@@ -32,7 +32,7 @@
 KT = 0.8
 
 
-class TestLBPressureACF:
+class TestLBPressureTensor:
```

Comment:
i think this testcase needs a lot more documentation

Reply:
After adding the references to equations, which parts of the test are not clear to you? I think, as is, there are sufficient comments that do in combination with naming of fuctions and variables explain what happens. However, if there are specific places that are not clear yet, I am happy to add more comments.